tests/dummy/app/styles/app.scss

```diff
@@ -18,7 +17,6 @@
     white-space: -o-pre-wrap;
   }
   code {
-    font-family: Monaco, Consolas, "Andale Mono", "DejaVu Sans Mono", monospace;
```

Comment:
hmm, this is apparently for code samples in the demo
do we really want to remove this?